app/sprinkles/account/config/default.php

```diff
@@ -162,7 +162,18 @@
         'check_username_request' => null,
         'password_reset_request' => null,
         'registration_attempt'   => null,
-        'sign_in_attempt'        => null,
+        'sign_in_attempt'        => [
```

Comment:
N.B.: The `sign_in_attempt` is `null` on purpose, as it's enabled only on production environment.